colossalai/inference/README.md

```diff
@@ -53,27 +64,39 @@ pip install -e .
 
 ### Requirements
 
-dependencies
+Install dependencies.
 
 ```bash
-pytorch= 1.13.1 (gpu)
-cuda>= 11.6
-transformers= 4.30.2
-triton
-# for install flash-attention
-flash-attention
+pip install -r requirements.txt
+pip install transformers==4.34.0
```

Comment:
Why not move this to requirements.txt

Reply:
Add requirements-infer.txt. Move the required package in requirements-infer.